vsintegration/src/FSharp.Editor/Navigation/GoToDefinitionService.fs

```diff
@@ -118,6 +125,8 @@ type internal FSharpGoToDefinitionService
                                                 | (:? FSharpEntity as symbol1), (:? FSharpEntity as symbol2) when x.IsFromDefinition ->
                                                     symbol1.DisplayName = symbol2.DisplayName
                                                 | (:? FSharpMemberOrFunctionOrValue as symbol1), (:? FSharpMemberOrFunctionOrValue as symbol2) ->
+                                                    let symbol1 = symbol1
+                                                    let symbol2 = symbol2
```

Comment:
I can remove this if I can't figure out the suave case. But it's here to facilitate debugging for now